rpc/src/v1/types/transaction.rs

```diff
@@ -69,6 +69,26 @@ pub enum Transaction {
         nonce: u64,
         hash: H256,
     },
+    #[serde(rename_all = "camelCase")]
+    AssetCompose {
+        network_id: NetworkId,
+        shard_id: ShardId,
+        world_id: WorldId,
+        nonce: u64,
+        metadata: String,
+        registrar: Option<PlatformAddress>,
+        inputs: Vec<AssetTransferInput>,
+        output: AssetMintOutput,
+    },
+    #[serde(rename_all = "camelCase")]
+    AssetDecompose {
+        network_id: NetworkId,
+        shard_id: ShardId,
+        world_id: WorldId,
```

Comment:
I guess shard_id and world_id are not needed.

Reply:
I removed it.